src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int NUMBER_OF_ELEMENTS = 3;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
```

Comment:
1) Let's check if `inputReport` is not null and throw an exception otherwise
2) We don't need to call `replaceAll()` method, because csv file has a certain structure
3) Move indexes to the constants